src/classification/__main__.py

```diff
@@ -2,24 +2,25 @@
 import sys
 import re
 
-# Start of importing the utilities module
-sys.path.insert(0, os.path.abspath("../utilities/"))
-import get_song_id as s_id
-# End of importing the utilities module
-
-from extractor.low_level_data_extractor import make_low_level_data_file
-from classifier.profile_data_extractor import get_classifier_data
-
 from pprint import pprint
 from tabulate import tabulate
-
 from multiprocessing import Pool
 
+import utilities.get_song_id as s_id
+from extractor.low_level_data_extractor import make_low_level_data_file
+from classifier.profile_data_extractor import get_classifier_data
+
 
 def process_data_and_extract_profiles(song_id, song_file, song_output_file):
+    # Create temp
+
+    #use temp
     make_low_level_data_file(song_file, song_output_file)
 
+    #use temp
```

Comment:
same comment as the other use temp, what does that mean, either remove it or make it more explanitory